unstructured/ingest/connector/elasticsearch.py

```diff
@@ -104,21 +105,52 @@ def _concatenate_dict_fields(self, dictionary, seperator="\n"):
         concatenated_values = seperator.join(values)
         return concatenated_values
 
+    @requires_dependencies(["elasticsearch"], extras="elasticsearch")
+    def _get_document(self):
+        from elasticsearch import Elasticsearch, NotFoundError
+
+        try:
+            # TODO: instead of having a separate client for each doc,
+            # have a separate client for each process
+            es = Elasticsearch(self.connector_config.url)
+            document = es.get(
+                index=self.connector_config.index_name,
+                id=self.document_meta.document_id,
+            )
+        except NotFoundError:
+            logger.error("Couldn't find document with ID: %s", self.document_meta.document_id)
+            return None
+        except Exception:
+            raise
```

Comment:
why this? if we just don't catch the exception here, this should happen automatically right?